lasagne/utils.py

```diff
@@ -202,7 +202,7 @@ def compute_norms(array, norm_axes=None):
             "Must specify `norm_axes`".format(array.ndim)
         )
 
-    norms = np.sqrt(np.sum(array**2, axis=sum_over))
+    norms = np.sqrt(np.sum(array ** 2, axis=sum_over))
```

Comment:
This one was completely unprompted, and a result of my compulsion to put spaces between all operators without thinking about it. I removed it from the PR.